coriolis/osmorphing/base.py

```diff
@@ -125,11 +125,13 @@ def _list_dir(self, chroot_path):
         return utils.list_ssh_dir(self._ssh, path)
 
     def _exec_cmd(self, cmd):
-        return utils.exec_ssh_cmd(self._ssh, cmd, self._environment)
+        get_pty = 'sudo ' in cmd
```

Comment:
I think we can just set the default value for ```get_pty``` to ```True```, and skip this check. We set it to ```True``` in most places, as almost all commands we run, require elevated privileges.